src/Interactive/CsiCore/project.json

```diff
@@ -54,7 +54,6 @@
     "System.Xml.XPath.XDocument": "4.3.0"
   },
   "frameworks": {
-    // We don't actually target netstandard1.6; this is to work around https://github.com/dotnet/roslyn/issues/12458
```

Comment:
@agocke We are not using a strict JSON parser; AFAIK the one we are using does allow comments.